features/host_can_view_navbar.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Host can view navbar and see "Become a Host" link 
+    As a Host 
+    In order to use the service
+    I would like to navigate the site
+
+  @javascript
+  Scenario: Visitor can see the landing page
+    When I visit the landing page
+    Then I should see a navigation bar
+    And I should see "Welcome Admin"
+    And I should see "BECOME A HOST" link 
+    And I should see "NEW LISTING" link
+    And I should see "SIGN UP" link
```

Comment:
If you see Welcome Admin, then you are already signed up and logged in, i think instead of seeing sign up, you should see log out?

Reply:
@zanenkn please add your comments to the relevant line of code

Reply:
Changed to just Welcome